content/codesprint-2024/contributions.md

```diff
@@ -0,0 +1,163 @@
++++
+fragment = "content"
+weight = 40
+background = "light"
+
+#title = "GeoStyler Code Sprint 2023"
++++
+
+## Contributions
+
+During the Code Sprint, we achieved the following:
+
+<img src="/images/geostyler-codesprint-2024-stats.png" alt="GeoStyler Code Sprint Stats" />
+
+<br />
+
+### GeoStyler ArcGIS Parser
+
+One of the probably biggest and hopefully most impactful achievements is the
+introduction of the [geostyler-lyrx-parser](https://github.com/geostyler/geostyler-lyrx-parser) which allows
+for parsing of ArcGIS Pro styles (.lyrx format)! The parser is based on the [GeoCat/bridge-style](https://github.com/GeoCat/bridge-style)
+and was now ported from Python into TypeScript and integrated into the GeoStyler GitHub organisation, so that
+the parser integrates seamlessly with all the other existing parsers! That means that it is now possible to convert
```

Comment:
```suggestion
for parsing of ArcGIS Pro styles (.lyrx format)! The parser is based on the [GeoCat/bridge-style](https://github.com/GeoCat/bridge-style)
and was ported from Python into TypeScript. It is now and integrated into the GeoStyler eco sytem, so that it integrates seamlessly with all the other existing parsers! That means that it is now possible to convert
```